src/components/pages/newReservation.js

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation } from 'react-router';
+import { createReservation } from '../../redux/reservation/thunk';
+import { fetchDoctors } from '../../redux/doctor/doctorSlice';
+import { selectStatus } from '../../redux/user/userSlice';
+import SideNav from '../navbar/SideNav';
+
+const NewReservation = () => {
```

Comment:
- [ ] When I try to create a new reservation after creating a new user, I does not work:

![image](https://github.com/tajulafreen/Final-capstone-Frontend/assets/57421823/db7678ba-fecf-4978-8925-1361134dec1e)

This is because upon creating a new user, the local storage is not populated:

![image](https://github.com/tajulafreen/Final-capstone-Frontend/assets/57421823/cd7051eb-93c8-4c11-82c9-1e73e9cd8272)

After creating a new user, I need to logout and login again, and it will work:

![image](https://github.com/tajulafreen/Final-capstone-Frontend/assets/57421823/ceb93f9b-cc05-4bf8-b3b6-ee189cccd4ae)

Kindly, fix this bug so your app works as intended. 👍